<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CClientMainWindow</class>
 <widget class="QMainWindow" name="CClientMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCS - Программа продавца</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Мероприятия</string>
        </property>
        <property name="icon">
         <iconset resource="clinet_res.qrc">
          <normaloff>:/clientImages/categoriesImages/imActions.png</normaloff>:/clientImages/categoriesImages/imActions.png</iconset>
        </property>
       </item>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget">
       <widget class="QWidget" name="page_actions">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="CClientActionsWidget" name="wActions" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CClientActionsWidget</class>
   <extends>QWidget</extends>
   <header>cclientactionswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="clinet_res.qrc"/>
 </resources>
 <connections/>
</ui>
